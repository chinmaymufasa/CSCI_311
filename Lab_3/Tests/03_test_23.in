2
132
5
2
58
5
2
91
5
2
57
5
2
4
5
1
5
-1