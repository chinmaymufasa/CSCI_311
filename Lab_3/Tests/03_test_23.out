132 
58 132 
58 91 132 
57 58 91 132 
4 57 58 91 132 
Not found
